### ISSUE TITLE
Extremely slow

### ISSUE THREAD

**Issue by madsbuch** [elixir-lsp/vscode-elixir-ls#286]
Type: <b>Bug</b>

This extension basically breaks the entire VS Code and is unbarably slow.

Activation time is 2000ms

Extension version: 0.12.0
VS Code version: Code 1.74.1 (Universal) (1ad8d514439d5077d2b0b7ee64d2ce82a9308e5a, 2022-12-14T10:33:40.793Z)
OS version: Darwin arm64 22.1.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Max (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|28, 52, 30|
|Memory (System)|64.00GB (3.81GB free)|
|Process Argv|. --crash-reporter-id 9ae0021c-5334-4811-900e-cf679f7676c7|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30577860
nodejswelcome1:30587005
3biah626:30602489
gswce1:30612156
iaj6b796:30613358
dbltrim-noruby:30604474
f6dab269:30613381
fim-prod:30623723

```

</details>

**Comment by VictorGaiva** [comment]
#285 seems to be about the same issue.

**Comment by pejrich** [comment]
Yeah, I'm seeing this too. Narrowed it down to elixir-ls. Snippets/autocomplete either fails to show up or when it does takes 5-10 seconds. Disabling elixir-ls brings it back to near immediate responsiveness.

**Comment by VictorGaiva** [comment]
@pejrich for now my recommendation is to downgrade the plugin to version 0.11

**Comment by lukaszsamson** [comment]
This should be resolved in v0.13.0. Please reopen if the issue persists
